src/translations/en.json

```diff
@@ -2988,15 +2988,15 @@
               "blueprint_config": "Blueprint Config"
             },
             "path": {
-              "choose": "Select a node on the left for more information.",
+              "choose": "Select a step on the left for more information.",
               "default_action_executed": "The default action was executed because no options matched.",
-              "no_further_execution": "This node was not executed and so no further trace information is available.",
-              "disabled_node": "This node was disabled and skipped during execution so no further trace information is available.",
+              "no_further_execution": "This step was not executed and so no further trace information is available.",
+              "disabled_step": "This step was disabled and skipped during execution so no further trace information is available.",
```

Comment:
Can we keep the same keys, so the translations are not lost?

Reply:
Yes, but translations were added only 2 weeks ago and are not in a release yet, do we need to keep the keys? 🤔

Reply:
They are probably already translates though, let me check

Reply:
When you change the key of a translation, the old and new key of the translation are kept in localise. So, if it is already translated, the translation has to be redone. If it isn't, the translation needs to be translated twice. Next time, keep the key, then the translation is kept for history and doesn't need to be translated once or twice.